ingredients_input/ingredients_input.php

```diff
@@ -38,6 +38,27 @@
     }
     ?>
     <div class="ingredients_image"></div>
+    <nav class="navbar navbar-expand-lg navbar-light bg-transparent d-flex">
+        <div class="container-fluid">
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
Here the same thing with the id in snake case

Reply:
This is from bootstrap. Changing it will break responsivness.